src/modules/pages/components/ExpenditurePage/constants.ts

```diff
@@ -54,6 +54,13 @@ export const initialValues = {
       { ...initalRecipient, key: nanoid() },
     ],
   },
+  streaming: {
+    startDate: { date: new Date() },
```

Comment:
With respect to setting the UTC time, could you not do it here? Then you wouldn't need to do those conversions onChange. 

Eg:

``` js

const nowUTC = () => {
  const date = new Date();
  const utc = new Date(
    date.getUTCFullYear(),
    date.getUTCMonth(),
    date.getUTCDate(),
    date.getUTCHours(),
    date.getUTCMinutes(),
  );
  return utc;
};
{
  startDate: { date: nowUTC() }
...
}

```

And remove lines 98-102 and 122-132 in `DatePicker`, and just have the component always reading from the form state (`value.date`).

Reply:
Since the "local UTC time" represents an incorrect moment in time, I think it makes more sense to do all the hacky conversion in the DatePicker component and only ever expose the correct Date object. I checked with Arren and we're not planning support for timezones other than UTC so this solution seemed like the most sensible one